src/main/java/noel/tasks/ToDos.java

```diff
@@ -0,0 +1,21 @@
+package noel.tasks;
+
+public class ToDos extends Task {
+
+    final String taskChar = "[T]";
```

Comment:
Same as your `taskChar` in your `Events` and `Deadlines` class, remember to capitalise constants :)